builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -452,3 +452,38 @@ def jsonParse(jsonString) {
     def nonLazyMap = new groovy.json.JsonSlurperClassic().parseText(jsonString)
     return nonLazyMap
 }
+
+def updateAclPermission(serviceId, userId) {
```

Comment:
Please move this code to a separate module.

Reply:
moved to scm-module